cmd/kpod/push.go

```diff
@@ -194,3 +194,20 @@ func pushImage(srcName, destName string, options pushOptions) error {
 	}
 	return nil
 }
+
+// pushImageForSave calls pushImage with the right pushOptions to save an
+// image into docker-archive or oci
+func pushImageForSave(src []string, dest string, store storage.Store) error {
+	opts := pushOptions{
+		SignaturePolicyPath: "",
+		Store:               store,
+	}
+
+	for _, image := range src {
```

Comment:
Should add a comment here stating that only one image is supported at this time.  Future pull requests will fix this.